values-schema.yaml

```
@@ -2579,7 +2579,7 @@ properties:
           - password
       hasCloudLB:
         default: false
-        description: 'Set this to true when an external LB exists or needs to be started (AWS ALB, Azure AppGW, Google Apigee). This will then be configured through ingress controllers. Expects existing LBs to terminate https. Currently this is only working correctly for Azure, and not for AWS and Google. AWS is close to completion.'
+        description: Set this to true when a cloud-specific ingress controller, which terminates TLS, is required. Otherwise, the Nginx Ingress Controller is deployed to the cluster.'
```

this is not correct, as nginx is always deployed. I suggest to stick to this:
```suggestion
        description: "Set this to true when a cloud specific ingress controller is wanted/installed that terminates TLS and forwards traffic unencrypted. Otherwise Otomi's Nginx controller will create a loadbalancer that will take care of this."
```